test/v1/services/ListBsasServiceSpec.scala

```diff
@@ -37,7 +37,7 @@ class ListBsasServiceSpec extends UnitSpec {
   private val taxYear = DesTaxYear("2019-20")
   private val incomeSourceIdentifier = Some("IncomeSourceType")
   private val identifierValue = Some("01")
-  private val correlationId = "X-123"
+  private implicit val correlationId = "X-123"
```

Comment:
You can just add this to the **ServiceSpec** like it was done in the ConnectorSpec and this class should extend ServiceSpec. At the moment it extends UnitSpec which should be changed.